dll/win32/shell32/lang/bg-BG.rc

```diff
@@ -94,7 +94,7 @@ CAPTION "Търсене на папка"
 FONT 8, "MS Shell Dlg"
 BEGIN
     DEFPUSHBUTTON "Добре", 1, 80, 176, 50, 12, BS_DEFPUSHBUTTON | WS_GROUP | WS_TABSTOP
```

Comment:
```suggestion
    DEFPUSHBUTTON "Добре", 1, 80, 176, 50, 12, WS_GROUP
```
These two removed styles are already in `DEFPUSHBUTTON`

Reply:
No don't do that! That suggestion is wrong @HBelusca . You would lose the tabstop then (due to the extended syle WS_GROUP being set here explicitly!). And that is wrong!

Reply:
Im okay with cleaning my code. However, I can't find those buttons to test how the navigation goes. Where are these?

Reply:
However @JoachimHenze is right too. For just a simple fix, we could clean and properly test all styles later in another PR and avoid delaying this one